CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # Change Log
 
+## [0.12.0](https://github.com/confluentinc/ksql/releases/tag/v0.12.0-ksqldb) (2020-09-01)
+
+### Features
+
+* add a serverStatus to ServerInfo and display the status in the CLI ([#6040](https://github.com/confluentinc/ksql/pull/6040)) ([1921d0e](https://github.com/confluentinc/ksql/commit/1921d0eaf7f9005014cb5c1158fa12e29f1c2fd2))
+* add config for suppress buffer ([#5969](https://github.com/confluentinc/ksql/pull/5969)) ([53b0d4d](https://github.com/confluentinc/ksql/commit/53b0d4d7e00fae6443cc4dc9039b9dee9ab805d2))

```

Comment:
#6094 disabled this - cc @derekjn I'm not sure how we want to message this but it might be valuable to not include it in the changelog. When we do enable suppression by default there will be more messaging explicitly around that

Reply:
Agreed, let's leave this out for now until it's enabled by default.

Reply:
If we leave this one out, then we should probably exclude the other feature bullets that reference suppress, right?

Reply:
Removed, along with other suppress commits.